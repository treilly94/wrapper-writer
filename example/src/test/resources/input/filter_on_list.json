{"col1": "cheese", "col2":2, "targetCol": 30},
{"col1": "ham", "col2":9, "targetCol": 67},
{"col1": "crackers", "col2":5, "targetCol": 1},
{"col1": "bicuits", "col2":4, "targetCol": 18},
{"col1": "chocolate", "col2":4, "targetCol": 2},
{"col1": "apples", "col2":8, "targetCol": 5}