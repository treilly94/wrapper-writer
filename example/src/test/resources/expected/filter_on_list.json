{"col1": "cheese", "col2":2, "targetCol": 30},
{"col1": "ham", "col2":9, "targetCol": 67},
{"col1": "bicuits", "col2":4, "targetCol": 18}